dvc/project.py

```diff
@@ -1065,32 +1063,35 @@ def metrics_show(self,
 
         for branch, val in res.items():
             if all_branches or all_tags:
-                self.logger.info('{}:'.format(branch))
+                logger.info('{}:'.format(branch))
             for fname, metric in val.items():
-                self.logger.info('\t{}: {}'.format(fname, metric))
+                logger.info('\t{}: {}'.format(fname, metric))
 
         if res:
             return res
 
         if path:
-            msg = 'File \'{}\' does not exist'.format(path)
+            msg = 'file "{}" does not exist'.format(path)
```

Comment:
minor: should we use single quotes everywhere? like for example a few line below, wrap a message with double quotes to avoid escaping.

Reply:
Thanks for pointing that one, @shcheklein, I've _amended_ another commit where I try to normalize the use of quotes and escaping :)